CHANGES.md

```diff
@@ -12,6 +12,8 @@ Apollo 1.10.0
 * [Make Access Key Timestamp check configurable](https://github.com/ctripcorp/apollo/pull/3908)
 * [remove ctrip profile](https://github.com/ctripcorp/apollo/pull/3920)
 * [Remove spring dependencies from internal code](https://github.com/apolloconfig/apollo/pull/3937)
+* [Support export/import configs](https://github.com/apolloconfig/apollo/pull/3947)
```

Comment:
How abount `Support export/import configs` ---> `Support export/import configs by apollo env`?

Because we provide `export ` configs api before 1.10.0

Reply:
done